input/templates/my-account-change-password.hbs

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  {{> common/htmlhead}}
+</head>
+
+<body>
+  <div class="darkbg hidden"></div>
+  {{> common/header}}
+  <div class="my-account">
+    <div class="container">
+      <div class="my-account-title">
+        <span class="my-account-title-text icon-user">{{i18n "my-account:myAccount.title"}}</span>
+      </div>
+      <div class="row">
+        <div class="col-sm-9">
+          <div class="my-account-content">
+            <div class="row">
+              <div class="col-sm-3">
+                {{> myaccount/my-account-sidebar positionSeven=true}}
+              </div>
+              <div id="my-account-desktop-content" class="col-sm-9">
+                <div class="change-password">
+                  <div class="change-password-wrapper">
+                    <div class="personal-details-title">
+                      <span>{{i18n "my-account:changePassword.changePasswordTitle"}}</span>
+                      <span class="pull-right required-text">{{i18n "main:form.required"}}*</span>
+                    </div>
+                    <form action="">
+                      <div class="row">
+                        <div class="col-sm-6">
+                          <div class="form-sections">
+                            <span class="form-labels">{{i18n "my-account:changePassword.labelTxt"}}*</span><br>
+                            <input class="form-inputs" type="text">
+                          </div>
+                        </div>
+                        <div class="col-sm-6">
+                          <div class="form-sections">
+                            <span class="form-labels">{{i18n "my-account:changePassword.confirmLabelTxt"}}*</span><br>
+                            <input class="form-inputs" type="text">
+                          </div>
+                        </div>
+                      </div>
+                    </form>
+                    <hr class="light-grey-hr">
+                    <div class="personal-details-edit-btn">
+                      <span>
+                        <button class="update-btn password-change-btn">{{i18n "my-account:changePassword.changeBtn"}}</button>
+                      </span>
+                    </div>
+                  </div>
+                  <div class="password-feedback-wrapper">
+                    <br /><br />
+                    <span>{{i18n "my-account:changePassword.feedbackMessage"}}</span>
```

Comment:
This is an interesting topic, because nobody has taken care of the messages, so we don't have really support for them in Sunrise. The only thing you can find from time to time in the templates are a placeholder for errors, like this one:
https://github.com/commercetools/commercetools-sunrise-theme/blob/master/input/templates/partials/myaccount/login-box.hbs#L15
So I'm happy that you put this one, but I would:
1) Put it where the error messages go, so on the top of the form.
2) Create a small template such as `form/global-errors.hbs`, named `global-messages.hbs` (in `form/global-messages.hbs`) and adapt it to look like a normal message (no failure feeling). The code will look like:
```
{{#each messages.globalMessages}}
    <div class="alert alert-info">{{message}}</div>
{{/each}}
```
3) Use it here.

You see you won't be using `{{i18n "my-account:changePassword.feedbackMessage"}}` in your code anymore, but don't remove it from the YAML file, since I will be using it from Sunrise app :)